src/constants/tooltips.tsx

```diff
@@ -87,6 +87,7 @@ export const ReloadFolderTooltip = () => (
 export const ResetFiltersTooltip = () => <HotkeyLabel text="Reset Filters" name="RESET_RESOURCE_FILTERS" />;
 export const SettingsTooltip = () => <HotkeyLabel text="Open Settings" name="TOGGLE_SETTINGS" />;
 export const ValidationTabTooltip = () => <HotkeyLabel text="View Validation" name="OPEN_VALIDATION_TAB" />;
+export const TutorialPageTooltip = () => <HotkeyLabel text="View Validation" name="OPEN_TUTORIAL_PAGE" />;
```

Comment:
Do we need this tooltip? I am assuming that the tutorial page won't be an option in the left menu

Reply:
@devcatalin 
Hi, Catalin, I will be later working on Getting Started as well, so should I remove the tooltip and the side menu option for good now or just hide it, I got this review comment from Razvan regarding the fact that we do not need it, but if later I have to add it again, I see no point in removing it in that case, but if the next step will be actually implementing the cases when the page is visible, there is no point in keeping it.

What do you think guys?

Reply:
Hi Mariam, you can remove it from the sidebar, I don't think it will ever be a dedicated option on the left menu, we'll figure out a way to make it appear, maybe it will be from a button in the menu from top right of the application, not sure. 
So you can remove it from the left menu and the tooltip, we'll revisit it after we have the links ready.

Reply:
ok, thanks

Reply:
Done